packages/webpack/src/runtime/runtime.js

```diff
@@ -1,7 +1,6 @@
 /// <reference path="./lavamoat.d.ts" />
 /* global LAVAMOAT */
-/* global lockdown, harden, Compartment */
-
+/* global Compartment */
```

Comment:
Should this be `/// <reference types="ses" />`?

Reply:
I have no idea, I just took out the parts that were no longer relevant

Reply:
`/// <reference types="ses" />` used to be in the local d.ts but it was impossible to untangle the esm/cjs problems with loading types from ses from dependencies - I remember we both tried that over zoom and settled on what's there now. Happy to reopen one day.